### DISCUSSION TITLE
koyeb安装后端服务器为什么刮削不正常呢

### DISCUSSION THREAD

**Discussion by jamesvon1** [metatube-community/jellyfin-plugin-metatube#386] [General]
特意从GitHub拉取的最新版本，可是刮削基本停滞的，日志一直没有变化，隔一两个突然又开始刮削，忽然又自动停了，几乎无法正常使用。
docker安装方式也试过，就是像fellatiojapan挂美国梯子无法刮削，又没有好的日本节点。哪位大佬指点一下，谢谢了

**Comment by sply94928** [comment]
大佬解决了吗？我也是
